roles/notifications/files/notification-startup.service

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Notify Telegram channel, server has started running
```

Comment:
```suggestion
Description=Notify Telegram channel, server has started running
Type=oneshot
```

Reply:
Why would it need to use oneshot if the execution is controlled by a timer?